experiment/manage_release_versions.py

```diff
@@ -25,28 +25,34 @@
 
 """
 
+from __future__ import print_function
```

Comment:
This is not used. Should remove.

Reply:
@imkin this is for silencing a pylint error https://stackoverflow.com/questions/28694380/pylint-says-unnecessary-parens-after-r-keyword